Components/Button/button.css

```diff
@@ -0,0 +1,95 @@
+:root{
+  --medium-font-size: medium;
+ --bolder-font-weight:bolder;
+ --black:#000000;
+ --default-btn-bg:#201D4F;
+}
+
+.button-section {
+    margin-top: 125rem;
+}
+
+span {
+    font-weight: var(--bolder-font-weight);
+}
```

Comment:
why adding css to directly element?

Reply:
Now I have updated it by giving that element a Class 👍🏻